### ISSUE TITLE
[Fabric] DrawScreenEvent render error when Gamemode Screen is open

### ISSUE THREAD

**Issue by Nidre** [Keksuccino/FancyMenu#133]
Error spam In logs

**To Reproduce**
1. Go to server
2. When you OP click F3+F4 to open gamemode menu
3. See error

**Error Log**
```
[03:27:52] [main/INFO]: [STDOUT]: ################# ERROR [KONKRETE] #################
[03:27:52] [main/INFO]: [STDOUT]: Failed to invoke event!
[03:27:52] [main/INFO]: [STDOUT]: Method Name: onRenderPost
[03:27:52] [main/INFO]: [STDOUT]: Event Name: de.keksuccino.konkrete.events.client.GuiScreenEvent$DrawScreenEvent$Post
[03:27:52] [main/INFO]: [STDOUT]: ####################################################
[03:27:52] [main/INFO]: [STDOUT]: 
[03:27:52] [main/INFO]: [STDERR]: java.lang.reflect.InvocationTargetException
[03:27:52] [main/INFO]: [STDERR]: 	at jdk.internal.reflect.GeneratedMethodAccessor17.invoke(Unknown Source)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.konkrete.events.EventHandler.invokeEvent(EventHandler.java:115)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.konkrete.events.EventHandler.callEventsFor(EventHandler.java:102)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.class_757.handler$zko000$onRenderPost(class_757.java:1982)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.class_757.method_3192(class_757.java:616)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1523(class_310.java:1048)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1514(class_310.java:681)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:215)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[03:27:52] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
[03:27:52] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139)
[03:27:52] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:710)
[03:27:52] [main/INFO]: [STDERR]: 	at pro.gravit.launcher.***.*****(Unknown Source)
[03:27:52] [main/INFO]: [STDERR]: 	at pro.gravit.launcher.***.main(Unknown Source)
[03:27:52] [main/INFO]: [STDERR]: Caused by: java.lang.NullPointerException
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.fancymenu.menu.fancy.menuhandler.MenuHandlerBase.shouldCustomize(MenuHandlerBase.java:1139)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.fancymenu.menu.fancy.menuhandler.MenuHandlerBase.onRenderPost(MenuHandlerBase.java:963)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.fancymenu.menu.fancy.menuhandler.custom.MainMenuHandler.onRenderPost(MainMenuHandler.java:102)
[03:27:52] [main/INFO]: [STDERR]: 	... 20 more
[03:27:52] [main/INFO]: [STDOUT]: ################# ERROR [KONKRETE] #################
[03:27:52] [main/INFO]: [STDOUT]: Failed to invoke event!
[03:27:52] [main/INFO]: [STDOUT]: Method Name: onRenderPost
[03:27:52] [main/INFO]: [STDOUT]: Event Name: de.keksuccino.konkrete.events.client.GuiScreenEvent$DrawScreenEvent$Post
[03:27:52] [main/INFO]: [STDOUT]: ####################################################
[03:27:52] [main/INFO]: [STDOUT]: 
[03:27:52] [main/INFO]: [STDERR]: java.lang.reflect.InvocationTargetException
[03:27:52] [main/INFO]: [STDERR]: 	at jdk.internal.reflect.GeneratedMethodAccessor18.invoke(Unknown Source)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.konkrete.events.EventHandler.invokeEvent(EventHandler.java:115)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.konkrete.events.EventHandler.callEventsFor(EventHandler.java:102)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.class_757.handler$zko000$onRenderPost(class_757.java:1982)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.class_757.method_3192(class_757.java:616)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1523(class_310.java:1048)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1514(class_310.java:681)
[03:27:52] [main/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:215)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[03:27:52] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
[03:27:52] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139)
[03:27:52] [main/INFO]: [STDERR]: 	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
[03:27:52] [main/INFO]: [STDERR]: 	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:710)
[03:27:52] [main/INFO]: [STDERR]: 	at pro.gravit.launcher.***.*****(Unknown Source)
[03:27:52] [main/INFO]: [STDERR]: 	at pro.gravit.launcher.***.main(Unknown Source)
[03:27:52] [main/INFO]: [STDERR]: Caused by: java.lang.NullPointerException
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.fancymenu.menu.fancy.menuhandler.MenuHandlerBase.shouldCustomize(MenuHandlerBase.java:1139)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.fancymenu.menu.fancy.menuhandler.MenuHandlerBase.onRenderPost(MenuHandlerBase.java:963)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.fancymenu.menu.fancy.menuhandler.custom.MainMenuHandler.onRenderPost(MainMenuHandler.java:102)
[03:27:52] [main/INFO]: [STDERR]: 	at de.keksuccino.fancymenu.menu.fancy.menuhandler.custom.MoreRefinedStorageMainHandler.onRenderPost(MoreRefinedStorageMainHandler.java:45)
[03:27:52] [main/INFO]: [STDERR]: 	... 20 more
```

**Basic Informations (please complete the following information):**
 - OS: Win10
 - Konkrete_fabric_1.1.2_MC_1.16.3-1.16.5
 - Fancymenu_fabric_1.7.0_MC_1.16.3-1.16.5
 - Fabric 1.16.5
 - Active 163 mods: fermion@2.1.191, fabric-keybindings-v0@0.2.1+ca58154a7d, roughlyenoughitems-api@5.9.1, dynamicsoundfilters@1.2.0+1.16.5, kubejs@1605.3.5-build.155, notenoughcrashes@3.0.0, fabricloader@0.11.1, crowdin-translate@1.2, satin@1.5.1, indrev@1.7.15-BETA, cloth-durability-bar-api-v1@1.4.9, fabric-dimensions-v1@2.0.2+ca58154a7d, java@11, mechanix@3.3.0, megane@3.1.2+1.16.4, cotton-client-commands@1.0.1+1.16-rc1, fabric-game-rule-api-v1@1.0.6+ca58154a7d, fermion-modkeys@1.10.218, fabric-api-base@0.2.0+ab87788d3a, astromine-discoveries@1.11.5+fabric-1.16.5, reborncore@4.6.4+build.117, eldritch_mobs@1.6.6, xaeroworldmap@1.11.10, piglib@1.0.1, fiber@0.23.0-2, org_joml_joml@1.10.0, fabrication@1.3.1, randompatches@2.1.9-fabric, roughlyenoughresources@2.1.3, maybe-data@1.0.0-1.16.2, cardinal-components-level@2.7.10, org_jetbrains_kotlin_kotlin-stdlib@1.4.0, com_moandjiezana_toml_toml4j@0.7.2, astromine@1.11.5+fabric-1.16.5, terraform-dirt-api-v1@1.1.0+build.4, astromine-transportations@1.11.5+fabric-1.16.5, waila@1.9.22, fabric-blockrenderlayer-v1@1.1.5+ca58154a7d, astromine-foundations@1.11.5+fabric-1.16.5, terraform-wood-api-v1@1.0.0+build.4, chat_heads@0.2.2+1.16.2, org_jetbrains_kotlin_kotlin-stdlib-jdk7@1.4.0, libblockattributes_items@0.8.5, fabric@0.30.0+1.16, spatialharvesters@0.10.6, fabric-networking-v0@0.3.2+ca58154a7d, pling@1.2.0, konkrete@1.1.2, wrenchable@1.0.0+1.16.1, techreborn@3.7.3+build.192, fabric-screen-handler-api-v1@1.1.1+ca58154a7d, modmenu@1.14.15, blur@2.0.0, fabric-tool-attribute-api-v1@1.2.6+ca58154a7d, org_jetbrains_kotlin_kotlin-stdlib-jdk8@1.4.0, roughlyenoughitems-runtime@5.9.1, terraform-tree-api-v1@1.0.1+build.4, cloth-client-events-v0@1.4.9, fabric-events-interaction-v0@0.4.2+ca58154a7d, fabric-crash-report-info-v1@0.1.3+ca58154a7d, fabric-language-kotlin@1.4.0+build.1, patchouli@1.16.4-48-FABRIC, fabric-key-binding-api-v1@1.0.2+ca58154a7d, xaerominimapfair@21.1.1, cloth-config2@4.8.3, org_jetbrains_kotlinx_kotlinx-coroutines-jdk8@1.3.9, amecs@1.3.3+mc.1.16.3, fabric-resource-loader-v0@0.4.2+ca58154a7d, megane-runtime@3.1.2+1.16.4, org_jetbrains_kotlin_kotlin-reflect@1.4.0, antighost@1.16.4-fabric0.25.1-1.1.3, cardinal-components-entity@2.7.10, magna@1.4.3-1.16.2, libblockattributes_fluids@0.8.5, architectury@1.5.104, astromine-technologies@1.11.5+fabric-1.16.5, fabric-registry-sync-v0@0.7.4+ca58154a7d, fabric-commands-v0@0.2.2+ca58154a7d, cardinal-components-base@2.7.10, adorn@1.13.0+1.16.5, fabric-mining-levels-v0@0.1.3+ca58154a7d, fabric-networking-api-v1@1.0.1+ca58154a7d, krypton@0.1.1, charm@2.2.2, rpg-hud@3.8.1, fabric-models-v0@0.2.1+ca58154a7d, fabric-item-groups-v0@0.2.3+ca58154a7d, org_jetbrains_annotations@20.0.0, bedrockify@0.2.3, terraform-shapes-api-v1@1.0.0+build.4, fabric-renderer-api-v1@0.4.1+ca58154a7d, terrestria@2.1.6, megane-api@3.1.2+1.16.4, rhino@1.7.13, libcd@3.0.3+1.16.3, cardinal-components-block@2.7.10, mambience@2.1.0+1.16.4, fabric-structure-api-v1@1.1.4+ca58154a7d, blade@2.0.0, fabric-containers-v0@0.1.10+ca58154a7d, lithium@0.6.1, fabric-object-builder-api-v1@1.9.3+ca58154a7d, redbits@1.7.0, libblockattributes@0.8.4, terraform-config-api-v1@1.0.0+build.4, dual_riders@1.0.5, appliedenergistics2@8.1.0-alpha.13, fabric-rendering-data-attachment-v1@0.1.5+ca58154a7d, terraform-surfaces-api-v1@1.0.0+build.4, fabric-textures-v0@1.0.6+ca58154a7d, cardinal-components-item@2.7.10, com_electronwill_night-config_core@3.6.3, connected_block_textures@0.1.3+1.16, fabric-rendering-fluids-v1@0.1.13+ca58154a7d, starlight@0.0.3, chesttracker@0.7.11, fabric-lifecycle-events-v1@1.2.1+ca58154a7d, cardinal-components-chunk@2.7.10, sodium@0.1.1-SNAPSHOT, fabric-renderer-registries-v1@2.2.1+ca58154a7d, cardinal-components-util@2.7.10, fabric-loot-tables-v1@1.0.2+ca58154a7d, dynamicfps@2.0.1, fabric-particles-v1@0.2.4+ca58154a7d, autoconfigtoml@1.0.0-fabric, ae2ao@6.0.3, libblockattributes_core@0.8.5, terraform-biome-builder-api-v1@1.0.0+build.4, fabric-networking-blockentity-v0@0.2.8+ca58154a7d, roughlyenoughitems-default-plugin@5.9.1, minecraft@1.16.5, fabric-command-api-v1@1.0.10+ca58154a7d, jankson@3.0.1+j1.2.0, cloth-basic-math@0.5.1, amecsapi@1.1.3+mc1.16.5, fabric-screen-api-v1@1.0.0+c045166c7d, fabric-renderer-indigo@0.4.4+ca58154a7d, smoothboot@1.16.4-1.4.0, com_electronwill_night-config_toml@3.6.3, fabric-item-api-v1@1.2.1+ca58154a7d, roughlyenoughitems@5.9.1, libgui@3.3.2+1.16.4, fabric-entity-events-v1@1.0.3+ca58154a7d, fabric-rendering-v0@1.1.2+ca58154a7d, fabric-rendering-v1@1.5.1+ca58154a7d, terraform-overworld-biome-extensions-api-v1@1.0.0+build.4, polymorph@0.0.8-1.16.4, autoconfig1u@3.3.1, megane-impl@3.1.2+1.16.4, completeconfig@0.11.0, fabric-content-registries-v0@0.2.1+ca58154a7d, fabric-tag-extensions-v0@1.1.1+ca58154a7d, cardinal-components-world@2.7.10, fancymenu@1.7.0, indium@1.0.0, fabric-biome-api-v1@3.1.1+ca58154a7d, inmis@2.1.1-1.16.4, cloth-datagen-api-v1@1.4.9, trade@1.1.3, fabric-events-lifecycle-v0@0.2.1+ca58154a7d, astromine-core@1.11.5+fabric-1.16.5, fabric-object-builders-v0@0.7.2+ca58154a7d, team_reborn_energy@0.1.1

**Comment by Keksuccino** [comment]
Hi! Thank you for your issue report.
Could you send me your complete latest.log file (located at `.minecraft/logs/latest.log`, please?

**Comment by Nidre** [comment]
[lates2t.log](https://github.com/Keksuccino/FancyMenu/files/5898368/lates2t.log)

And my friend`s client don`t start with errors spam it`s his log:
[late1st.log](https://github.com/Keksuccino/FancyMenu/files/5898371/late1st.log)

**Comment by Keksuccino** [comment]
Gamemode screen bug fixed in FancyMenu [Fabric] v1.7.1!

Your friend's problem looks more like a mod conflict.
If he's still unable to start the game with v1.7.1, please open a new issue.

Thank you for your report!
